prov/util/src/util_attr.c

```diff
@@ -299,31 +295,38 @@ int ofix_getinfo(uint32_t version, const char *node, const char *service,
 		 const struct fi_info *hints, ofi_alter_info_t info_to_core,
 		 ofi_alter_info_t info_to_util, struct fi_info **info)
 {
-	struct fi_info *core_info, *util_info, *cur, *tail;
+	struct fi_info *core_info, *base_info, *util_info, *cur, *tail;
 	int ret;
```

Comment:
I believe we need to initialize ret to -FI_ENODATA now.  ofi_check_info() can skip all fi_info structures, leaving ret uninitialized.